.gitignore

```diff
@@ -12,3 +12,5 @@ README_OLD.md
 conf/ds-discover-environment.yaml
 conf/ds-discover-local.yaml
 setup_project.sh
+/deploy2Devel11.sh
+/.gitignore
```

Comment:
Please don't add .gitignore to .gitignore: It is shared between developers.

Reply:
hhmm. didnt know I did. fixed.